.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ jobs:
             matrix:
                 php-version: [
                         '7.4',
+                        '8.0',
```

Comment:
As discussed - we don't need to add support of PHP 8.0 here. Every project should use only min and max supported version of PHP

Reply:
Thanks ++